railties/lib/rails/tasks/initializers.rake

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
+require "rails/command"
+require "active_support/deprecation"
+
 desc "Print out all defined initializers in the order they are invoked by Rails."
-task initializers: :environment do
```

Comment:
I got rid of loading the environment here since we are loading it within the `Rails::Command`. My assumption is that requiring it both places would load it twice which is unnecessary, but please correct me if I'm wrong here .

The only down side of getting rid of it here is that our `ActiveSupport::Deprecation.behavior` will not be taken into consideration and therefore the warning is outputted to `stderr` instead of following whatever the user set up in their app. Please let me know if you feel strongly about keeping the environment loading here.